docs/integrations/sources/postgres.md

```diff
@@ -145,6 +135,16 @@ Please note that:
 
 The UI currently allows selecting any tables for CDC. If a table is selected that is not part of the publication, it will not replicate even though it is selected. If a table is part of the publication but does not have a replication identity, that replication identity will be created automatically on the first run if the Airbyte user has the necessary permissions.
 
+#### 5. Create replication slot
+
+Next, you will need to create a replication slot. Here is the query used to create a replication slot called `airbyte_slot`:
```

Comment:
```suggestion
Next, you will need to create a replication slot. It's important to create the publication first (as in step 4) before creating the replication slot. Otherwise, you can run into exceptions if there is any update to the database between the creation of the two.

Here is the query used to create a replication slot called `airbyte_slot`:
```